lib/kitchen/directions/bake_chapter_key_concepts.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module Kitchen
+  module Directions
+    # Bake directions for eoc key concepts
+    #
+    module BakeChapterKeyConcepts
+      def self.v1(chapter:, metadata_source:)
+        metadata_elements = metadata_source.children_to_keep.copy
+
+        fix_titles = chapter.pages.map do |page|
+          key_concepts = page.search('section.key-concepts')
+          next if key_concepts.none?
+
+          key_concepts.search('h3').trash
+          key_concepts.map do |key_concept|
+            id = key_concept.id.split('fs-id')
+            key_concept.prepend(child:
+              <<~HTML
+                <a href="##{id[0]}0">
```

Comment:
I wonder if this is always supposed to end in `0` and if so what it links to?  Seems a little weird.